flac.md

```diff
@@ -38,21 +38,29 @@ In addition, FLAC specifies a metadata system, which allows arbitrary informatio
 
 # Definitions
 
-Many terms like "block" and "frame" are used to mean different things in different encoding schemes. For example, a frame in MP3 corresponds to many samples across several channels, whereas an S/PDIF frame represents just one sample for each channel. The definitions we use for FLAC follow. Note that when we talk about blocks and subblocks we are referring to the raw unencoded audio data that is the input to the encoder, and when we talk about frames and subframes, we are referring to the FLAC-encoded data.
+- **Block**: A (short) section of linear pulse-code modulated audio, with one or more channels.
 
-- **Block**: One or more audio samples that span several channels.
+- **Subblock**: All samples within a corresponding block for 1 channel. One or more subblocks form a block, and all subblocks in a certain block contain the same number of samples.
 
-- **Subblock**: One or more audio samples within a channel. A block contains one subblock for each channel, and all subblocks contain the same number of samples.
+- **Frame**: A frame header plus one or more subframes. It encodes the contents of a corresponding block.
 
-- **Blocksize**: The number of samples in any of a block's subblocks. For example, a one second block sampled at 44.1 kHz has a blocksize of 44100, regardless of the number of channels.
+- **Subframe**: An encoded subblock. All subframes within a frame code for the same number of samples. A subframe MAY correspond to a subblock, else it corresponds to either the addition or subtraction of two subblocks, see [section on interchannel decorrelation](#interchannel-decorrelation).
 
-- **Frame**: A frame header plus one or more subframes.
+- **Blocksize**: The total number of samples contained in a block or coded in a frame, divided by the number of channels. In other words, the number of samples in any subblock of a block, or any subframe of a frame. This is also called **inter-channel samples**.
```

Comment:
Should the writing be unified either `inter-channel` or `interchannel`?

Reply:
@retokromer - thank you for noticing and asking. Yes, spelling and punctuation should be as consistent as we can make it. 

If people notice inconsistencies in our documents, please add a PR to fix them, because eventually, the RFC Editor will ask about them ("eventually" means "during AUTH48"), and the listed authors will have to approve bulk text changes at the last minute.